privacy-vn.md

```diff
@@ -0,0 +1,11 @@
+# Chính Sách Về Quyền Riêng Tư
+## Thông Tin Nào Được Thu Thập?
+Tiện ích NhacCuaTui Scrobbler sự dụng API Last.fm để lấy được xác thưc. Kết quả là tên người dùng và một mật khẩu đặc biệt cho sự dụng.
+
+Tiện ích chỉ chạy được trên các trang từ nhacuatui.com, va tìm kiếm tên bài hát và tên nghệ sĩ của bài đang nghe.
```

Comment:
There is a missing `c` in `nhacuatui.com`.

Also does `va` need to be changed to `và`?